app/frontend/wallet/shelley/helpers/addresses.ts

```diff
@@ -29,6 +30,15 @@ export const encodeAddress = (address: Buffer): Address => {
   return bech32.encode(addressPrefix, address)
 }
 
+// TODO: we might want to add this to cardano-crypto.js
+export const encodeAssetFingerprint = (policyIdHex: HexString, assetNameHex: HexString): string => {
```

Comment:
Why is it located in the `address` folder? Is fingerprint some kind of address for tokens or what is actually its purpose? :thinking:

Reply:
Good point, "addresses" is pretty outdated name, you are right to we should divide the file into multiple helper files since it already contains non-address related stuff such as `xpub2ChainCode`.  I ll create an issue for it.